torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -162,7 +162,9 @@ def maybe_increase_iter(self, bucket):
             )
 
 
-def powerSGD_hook(state: PowerSGDState, bucket) -> torch.futures.Future:
```

Comment:
Can you also add the typing for another hook batched_powerSGD_hook in the same file?